tests/utils/test_common.py

```diff
@@ -20,6 +20,14 @@ def test_raise_warning():
     assert record[0].message.args[0] == "My warning."
 
 
+def test_raise_future_warning():
+    with pytest.warns(FutureWarning) as record:
+        raise_warning("My future warning.")
```

Comment:
we'll have to emit a `FutureWarning`, or else the test fails
```suggestion
        raise_warning("My future warning.", category=FutureWarning)
```